integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -730,6 +730,20 @@ def write_partitions(spark, table_path):
         conf={}
     )
 
+# Test to avoid regression on a known bug in Spark. For details please visit https://github.com/NVIDIA/spark-rapids/issues/8693
+@pytest.mark.parametrize('with_x_session', [with_gpu_session, with_cpu_session])
+def test_hive_timestamp_value(with_x_session, spark_tmp_table_factory):
+    def create_table(spark):
+        tmp_input = spark_tmp_table_factory.get()
+        spark.sql(f"""CREATE TABLE {tmp_input} STORED AS PARQUET """ +
+        " AS SELECT CAST('2015-01-01 00:00:00' AS TIMESTAMP) as t; ")
+        return tmp_input
+
+    def read_table(path):
+        return lambda spark: spark.sql(f"""SELECT * FROM {path}""")
+
+    input_table = with_x_session(create_table)
```

Comment:
This doesn't actually test that the GPU write is correct, just that the CPU and GPU agree on how to read what the GPU wrote, incorrect or not.  This should use something like assert_gpu_and_cpu_writes_are_equal_collect to test the write path produces correct data and assert_gpu_and_cpu_are_equal_collect on the CPU-written data to verify the reads are correct.  Could also do assert_gpu_and_cpu_are_equal_collect on the GPU-written data to verify GPU can correctly read what it wrote.